README.md

```diff
@@ -168,6 +169,15 @@ GROUP BY
   <time_column>
 ```
 
+## Caching
+### Snowflake caching
+
+Snowflake caches queries with the same footprint / hash in its own query-cache. The utilization of the snowflake cache will optimize your query in speed and costs. Since a Grafana query mostly has a now() component the cache will never be used as the hash is changing with every queries.
+To create more queries with the same hash two macros `$__timeRoundFrom(d)` and `$__timeRoundTo(d)` will create truncated timestamps. They are truncated to the next multiple of d. Grafana will ask data in a wider window and the query can use the snowflake query-cache.
+This is no problem for timeseries charts as a grafana timeseries chart cuts it's x-Axis to the panel time window. Slices before and after will not be rendered. Take care that the additional data is part of the resulting dataset. If a table is displayed the whole result will be presented and it could be slightly out of the original asked time-frame.
+
+![Caching Example](./img/timeRound.png)
```

Comment:
I believe the goal of the plugin isn’t to dive deep into how Snowflake’s cache works. 
Instead of adding a schema, could you please just include a link to the official documentation ?
https://docs.snowflake.com/en/user-guide/querying-persisted-results#retrieval-optimization

Reply:
Of course! We could even delete this part completely. I just thought it could be a good idea to describe the value of the both macros. But yes i agree that we flood the readme with this kind of information. Shall we delete it completely or rework into an extra file or shorten with the link? Let me know, I will have a look!

Reply:
you can just remove the image, shorten a little bit the text and put the link to the Snowflake documentation.